### ISSUE TITLE
padrino Command Requires Gemfile to Run

### ISSUE THREAD

**Issue by sshaw** [padrino/padrino-framework#2072]
One can no longer generate a project -or even exec `padrino -v`- in versions >= 0.12.8.

This was introduced by 89edc6d (0.12.8) in an effort to address #2013.

Example:
```
/tmp >gem list padrino

*** LOCAL GEMS ***

padrino (0.13.3.1)
padrino-admin (0.13.3.1)
padrino-cache (0.13.3.1)
padrino-core (0.13.3.1)
padrino-gen (0.13.3.1)
padrino-helpers (0.13.3.1)
padrino-mailer (0.13.3.1)
padrino-support (0.13.3.1)
/tmp >padrino -v
/Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/gems/bundler-1.12.5/lib/bundler.rb:170:in `rescue in root': Could not locate Gemfile or .bundle/ directory (Bundler::GemfileNotFound)
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/gems/bundler-1.12.5/lib/bundler.rb:166:in `root'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/gems/bundler-1.12.5/lib/bundler.rb:74:in `bundle_path'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/gems/bundler-1.12.5/lib/bundler.rb:414:in `configure_gem_home_and_path'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/gems/bundler-1.12.5/lib/bundler.rb:60:in `configure'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/gems/bundler-1.12.5/lib/bundler.rb:121:in `definition'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/gems/padrino-core-0.13.3.1/lib/padrino-core/cli/binstub.rb:7:in `replace_with_binstub'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/gems/padrino-core-0.13.3.1/bin/padrino:3:in `<top (required)>'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/bin/padrino:23:in `load'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/bin/padrino:23:in `<main>'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/bin/ruby_executable_hooks:15:in `eval'
	from /Users/sshaw/.rvm/gems/ruby-2.2.3@padrino-13/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by nesquena** [comment]
@ujifgc

**Comment by ujifgc** [comment]
Fixed in 0.13.3.2 920dea631261a614e2a1ffb5b3314f5120cd6fb6 and 0.12.8.1 d3f4eb3bfc8d00ef37c31924a7340b0571a70683
